Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -169,7 +169,7 @@ public override AView FocusSearch(AView focused, [GeneratedEnum] FocusSearchDire
 			var element = Element as ITabStopElement;
 			int maxAttempts = 0;
 			var tabIndexes = element?.GetTabIndexesOnParentPage(out maxAttempts);
-			if (tabIndexes == null)
+			if (tabIndexes == null || tabIndexes.Count > 0)
```

Comment:
In this PR: https://github.com/xamarin/Xamarin.Forms/pull/6675
`GetTabIndexesOnParentPage` was changed to return an empty dictionary instead of null: https://github.com/xamarin/Xamarin.Forms/blob/f2e76c9c4b2a7d4e4b6b1f2fc6ab725f50aef6cf/Xamarin.Forms.Core/TabIndexExtensions.cs#L29
For this reason, this null check is no longer necessary. But, then validation you added then, shouldn't it be if the collection is empty?

Reply:
@robertomujica Let me know if we can help you with something.